browser/containers/Admin.tsx

```diff
@@ -140,15 +147,17 @@ export default class Admin extends Component<Props, State> {
             <h4>CCLAs</h4>
             <div className="list-group">
               <a href="#" onClick={this.setCLAList} className="list-group-item list-group-item-action">View CCLAs</a>
-              <a href="#" onClick={this.setCLAForm} className="list-group-item list-group-item-action">New CCLA</a>
+              <a href="#" onClick={this.setCLAFormTrue} className="list-group-item list-group-item-action">New CCLA</a>
             </div>
           </div>
 
           <div className="col-lg-10 mb-3">
             <div className="panel-body" key={this.state.key}>
-              {this.state.editContributionListBoolean ? (<EditContributionTable contributionList={this.state.contributionList} />) : this.renderTable(contributionList)}
+              {!this.state.approvalListBoolean && !this.state.editContributionListBoolean && !this.state.claTableBoolean && !this.state.claFormBoolean && <p>Select an option from the left.</p>}
```

Comment:
Whew, we'll want to re-work these at some point. 😅

Reply:
I thought this was cleaner than the standard ternary operation. :/